jsonrpcbase/main.py

```diff
@@ -363,20 +373,10 @@ class ParseError(JSONRPCError):
     message = 'Parse error'
 
 
-class InvalidIDType(JSONRPCError):
-    code = -32600
-    message = 'Invalid type for the `id` field'
-
-
-class InvalidParamsType(JSONRPCError):
-    code = -32600
-    message = 'Invalid type for the `params` field'
-
-
 class InvalidRequestError(JSONRPCError):
     """The received JSON is not a valid JSON-RPC Request."""
     code = -32600
-    message = 'Invalid request'
+    message = 'Invalid Request'
```

Comment:
Very nitpicky, but this is to exactly match the error messages listed in the spec https://www.jsonrpc.org/specification#error_object